### PR TITLE
[142] Develop a Contact Page using HTML

### PR DESCRIPTION
**Pull request by Kifaayah**
image

### CODE REVIEW

File: submissions/jimohKafayat/html-forms/contactform.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <input type="radio" name="gender" value="male" id="male"><br>
+        <label for="female">Female</label>
+        <input type="radio" name="gender" value="female" id="female"><br><br>
+		<label for="email">Email:</label><br>
+		<input type="text" name="email" id="email" placeholder="Eg. Example@gmail.com"><br><br>
+		<label for="phonenumber">Phone Number</label><br>
+		<input type="text" name="phonenumber" id="phonenumber" placeholder="enter your phonenumber"><br><br>
+
+
+
```
Comment (Ireti203) [inline submissions/jimohKafayat/html-forms/contactform.html:29]: Remove whitespaces

File: submissions/jimohKafayat/html-forms/contactform.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		</select><br><br>
+
+
+		<label for="message">Message:</label><br>
+		<textarea name="message" id="message" cols="70" rows="20"></textarea>
+		<input type="submit" name="submit" value="Submit">
+
+
+		
+
```
Comment (Ireti203) [inline submissions/jimohKafayat/html-forms/contactform.html:48]: Same here

File: submissions/jimohKafayat/html-forms/contactform.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+		<select name="subject" id="subject">
+			<option value="other">Other</option>
+			<option value="purchase">Purchase</option>
+			<option value="enquiry">Enquiry</option>
+			<option value="complaints">Complaints</option>
+			<option value="comment">Comments</option>
+			<option value="suggestions">Suggestions</option><br><br>
+		</select><br><br>
+
+
```
Comment (Ireti203) [inline submissions/jimohKafayat/html-forms/contactform.html:41]: Remove whitespaces

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by Ireti203** [comment]
You are to attach the screenshot of the page on your browser. Not screenshot of github...

File: submissions/jimohKafayat/HTML/productpage.html [hunk 4]
```diff
@@ -10,7 +10,9 @@ <h2><em>KAHF KOLLECTIONS</e></h2>
 
 <nav>
 	<p>CATEGORIES</p>
-	<ol>y
+	<ol>
+		<a href="../html-forms/contactform.html">Contact Us</a>
```
Comment (adaugochi) [inline submissions/jimohKafayat/HTML/productpage.html:14]: This navigation links you added here is for your next task. if you add it now you won't have anything to do for the next task and you won't be able to proceed. Kindly remove them for now

File: submissions/jimohKafayat/html-forms/contactform.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Contact Page</title>
+</head>
+	<meta charset="utf-8">
+<body>
+	    <a href = "../HTML/productpage.html"> Product Page </a>
+        <a href = "contactform.html"> Contact Us </a>
```
Comment (adaugochi) [inline submissions/jimohKafayat/html-forms/contactform.html:9]: Same here too

File: submissions/jimohKafayat/html-forms/contactform.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Contact Page</title>
+</head>
+	<meta charset="utf-8">
+<body>
+	    <a href = "../HTML/productpage.html"> Product Page </a>
```
Comment (adaugochi) [inline submissions/jimohKafayat/html-forms/contactform.html:8]: And here

File: submissions/jimohKafayat/HTML/productpage.html [hunk 7]
```diff
@@ -10,7 +10,9 @@ <h2><em>KAHF KOLLECTIONS</e></h2>
...
+		<a href="productpage.html">Product Page</a>
```
Comment (adaugochi) [inline submissions/jimohKafayat/HTML/productpage.html:15]: same here

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**